zerver/lib/webhooks/common.py

```diff
@@ -75,20 +77,21 @@ def check_send_webhook_message(
     user_profile: UserProfile,
     topic: str,
     body: str,
+    stream_id: Optional[int] = REQ(converter=to_non_negative_int, default=None),
     stream: Optional[str] = REQ(default=None),
```

Comment:
Do you think the right approach is to have a different parameter name, or to instead determine the type based on whether we're provided an integer or a string?

Reply:
I think an implicit logic here is that using "stream" will create a new stream when it doesn't exist, while "stream_id" won't. I think it might be helpful to differentiate them.

Reply:
Where do we create a stream if you try to send to a stream name that doesn't exist?  I wouldn't expect Zulip to have that behavior.

If I remember correctly that Zulip only creates streams on subscribe attempts, I think these become two different ways of encoding the stream to send to.

We'd end up with the API being very similar to this: https://zulip.com/api/send-message#parameter-to

Reply:
I just found that I confused the stream creation behavior of `check_send_webhook_message` with a test method called `check_webhook`. Then I think using `stream` as the parameter should be feasible.